src/subscriber.ts

```diff
@@ -852,12 +863,10 @@ export class Subscriber extends EventEmitter {
    * @private
    */
   async nackWithResponse(message: Message): Promise<AckResponse> {
-    const ackSpan = tracing.PubsubSpans.createReceiveResponseSpan(
```

Comment:
Are these ack spans being removed completely?

I think we still want them, but in addition to the events.

Reply:
The screenshots I saw seemed to imply that we're removing them in favour of the events, but again I might be misunderstanding. It might not be a bad idea to consolidate all of those specs just so they're in one place and up to date.

Reply:
The new otel linking design should be what our implementations are based off of. However, it's slightly out of date given the recent working group discussions.

With that said, @alevenberg is right in that we want both events and spans. These events live on the main subscribe trace (name TBD), while the span is created for the batch ack/nack/modack operation in its own trace.

Reply:
Oh, I see what you mean - yeah, good catch, I hadn't added the batch receive for subscriber yet, only for publish.